src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+:root {
+  --font-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  color: var(--font-color);
+}
+
+.main_header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  line-height: 60px;
+}
+
+.moyo_logo {
+  align-items: center;
+  height: 40px;
+  width: 40px;
+}
+
+.navigation {
+  display: block;
+  text-align: center;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_item {
+  text-decoration: none;
+  margin-right: 20px;
```

Comment:
Try usign margin-left instead

Reply:
Hi @alysonvilela now it passed in the tests.
I made the changes you told me and also removed the margin-left from the first-child.
Thanks!